cmd/common/registry.go

```diff
@@ -0,0 +1,342 @@
+// Copyright 2022-2023 The Inspektor Gadget authors
```

Comment:
In this review, I tried to follow the creation of the commands and their parameters. This is what I found:

- Runtime: 
  - Has only general parameters added to the root command. `local` does have any. Example of this for the future `remote` runtime?
- Operator:
  - Has general parameters added to the root command. An example of `local-gadget` is the [runtime](https://github.com/inspektor-gadget/inspektor-gadget/blob/a8ba1b0b13816c8dee55e062024ae5a20ce2cac9/internal/operators/localmanager/localmanager.go#L74). I suppose that for `kubemanager` there will be all the `KubernetesConfigFlags` and so on.
  - Has per gadget params added to each gadget command. An example of `localmanager` is the [containername](https://github.com/inspektor-gadget/inspektor-gadget/blob/a8ba1b0b13816c8dee55e062024ae5a20ce2cac9/internal/operators/localmanager/localmanager.go#L88). I suppose that for `kubemanager` there will be ns, pod, etc.
- Gadget:
  - Has its own parameters. 
  - Some other parameters are added based on the gadget type. For instance, [interval, sort and max-rows](https://github.com/inspektor-gadget/inspektor-gadget/blob/a8ba1b0b13816c8dee55e062024ae5a20ce2cac9/pkg/gadgets/params.go#L40-L45).

Reply:
Yup, sounds right.

Example for the remote runtime params would be the endpoint, whether or not to use the kubernetes API server etc.